scripts/update-dependencies.ps1

```diff
@@ -32,9 +32,15 @@ $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 
 $appPath = "$PSScriptRoot\update-dependencies"
 
-# Restore the build scripts
-Write-Host "Restoring Build Script projects..."
-pushd $PSScriptRoot
+# Restore the build_projects
+Write-Host "Restoring Microsoft.DotNet.Cli.Build.Framework..."
+pushd $PSScriptRoot\..\build_projects\Microsoft.DotNet.Cli.Build.Framework
```

Comment:
please don't use ..\ to determine paths in scripts. This makes it really hard to refactor later. Use a `$RepoRoot`-relative path instead.